packages/plugins/src/hardhat/tasks.ts

```diff
@@ -1352,3 +1355,171 @@ export const listProjectsTask = async ({}, hre: HardhatRuntimeEnvironment) => {
 task(TASK_CHUGSPLASH_LIST_PROJECTS)
   .setDescription('Lists all projects that are owned by the caller.')
   .setAction(listProjectsTask)
+
+const claimOwnershipTask = async (
+  args: {
+    configPath: string
+    referenceName: string
+    silent: boolean
+  },
+  hre
+) => {
+  const { configPath, referenceName, silent } = args
+  const provider = hre.ethers.provider
+  const signer = provider.getSigner()
+
+  const spinner = ora({ isSilent: silent })
+  spinner.start('Checking project registration...')
+
+  const parsedConfig = loadParsedChugSplashConfig(configPath)
+
+  // Throw an error if the project has not been registered
+  if (
+    (await isProjectRegistered(signer, parsedConfig.options.projectName)) ===
+    false
+  ) {
+    errorProjectNotRegistered(
+      await getChainId(hre.ethers.provider),
+      hre.network.name,
+      configPath
+    )
+  }
+
```

Comment:
We should check for a couple more things here:
1. We should throw an error if the caller is not the project owner (via `getProjectOwnerAddress` in the core package)
2. We should throw an error if there is currently an active bundle (via `manager.activeBundleId()`)